python/ray/autoscaler/_private/monitor.py

```diff
@@ -160,7 +160,8 @@ def _initialize_autoscaler(self):
                 self.load_metrics,
                 prefix_cluster_info=self.prefix_cluster_info,
                 event_summarizer=self.event_summarizer,
-                prom_metrics=self.prom_metrics)
+                prom_metrics=self.prom_metrics,
+                disable_node_updaters=self.disable_node_updaters)
```

Comment:
I think this should be reverted (it is the old codepath)

Reply:
thanks for catching that